src/test/java/hw_jdi/pages/MetalsAndColorsPage.java

```diff
@@ -0,0 +1,184 @@
+    private List<WebElement> metalsOptions;
+
+    @FindBy(xpath = "//button[@type = 'button'][@aria-haspopup = 'true']")
+    private WebElement vegetablesMenu;
+
+    @FindBy(xpath = "//ul[@class = 'dropdown-menu'][@style]/li/a/input")
+    private List<WebElement> vegetablesCheckboxesInput;
+
+    @FindBy(xpath = "//ul[@class = 'dropdown-menu'][@style]/li/a/label")
+    private List<WebElement> vegetablesCheckboxes;
+
+    @FindBy(xpath = "//button[@id = 'submit-button']")
+    private WebElement submitButton;
+
+    @FindBy(xpath = "//ul[@class = 'panel-body-list results']/li")
+    private List<WebElement> results;
+
+
+    public void setSummaryValues(List<Integer> values) {
+        if (values != null) {
+            for (int i = 0; i < values.size(); i++) {
+                if (values.get(i) % 2 != 0) {
+                    oddRadios.get((values.get(i) / 2)).click();
+                } else
+                    evensRadios.get((values.get(i) / 2) - 1).click();
+            }
+        }
+    }
+
+    public void setElementsValues(List<String> elements) {
+        if (elements != null) {
+            for (int i = 0; i < elementsCheckboxes.size(); i++) {
+                for (int j = 0; j < elements.size(); j++) {
+                    if (elements.get(j).equals(elementsCheckboxes.get(i).getText())) {
+                        elementsCheckboxes.get(i).click();
+                    }
+                }
+            }
+        }
+    }
+
+    public void setColorValue(String color) {
+        if (color != null) {
+            colorsMenu.click();
+            for (int i = 0; i < colorsOptions.size(); i++) {
+                if (color.equals(colorsOptions.get(i).getText())) {
+                    colorsOptions.get(i).click();
+                }
+            }
+        }
+    }
+
+    public void setMetalValue(String metal) {
+        if (metal != null) {
+            metalsMenu.click();
+            for (int i = 0; i < metalsOptions.size(); i++) {
+                if (metal.equals(metalsOptions.get(i).getText())) {
+                    metalsOptions.get(i).click();
+                }
+            }
+        }
+    }
+
+    public void setVegetablesValues(List<String> vegetables) {
+        if (vegetables != null) {
+            vegetablesMenu.click();
+            for (int i = 0; i < vegetablesCheckboxes.size(); i++) {
+                for (int j = 0; j < vegetables.size(); j++) {
+                    if ("Vegetables".equals(vegetables.get(j))) {
+                        continue;
+                    }
+                    if (vegetables.get(j).equals(vegetablesCheckboxes.get(i).getText())) {
+                        vegetablesCheckboxes.get(i).click();
+                    }
+                }
+            }
+        }
+    }
+
+    public void clickSubmitButton() {
+        submitButton.click();
+    }
+
+
+    public void assertSummary(List<Integer> summary) {
+        if (summary != null) {
```

Comment:
You should never ever make an 'if' clause in assertion methods. What is going to happen in case if the 'summary' represents as null?